### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by Arch-Noize**
### Added contact form
- [ ] In this milestone we did the following :blush: 
- We add a contact section. :ok_hand: 
- We style the contact section according to the Figma design.
- We implemented form action. :100: 
- The form was sent successfully to Formspree. :rocket:

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -401,7 +397,62 @@ button:active {
...
+
+input:invalid {
+  box-shadow: 1px 8px 16px #de564130;
+}
+
+input:valid {
+  box-shadow: 1px 8px 16px #41de8330;
+}
+
+input:focus:invalid {
```
Comment (Danny-imo) [inline styles.css]: This line is not working as intended,try using  `select:focus`for semantic purposes

**Review by Danny-imo — DISMISSED** [review]
You are doing great ,just need a couple more changes

**Review by Zuhaib042 — COMMENTED** [review]
Hi @Arch-Noize @Danny-imo 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is pair-programming activity, but I cant see any commit from @Danny-imo. kindly consider getting actual code commits from danny, not just some changes in readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by osamaashraf6 — APPROVED** [review]
Hi @Arch-Noize 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
MinionsGIF (2)

## Highlights🎉🎉

- [ ] All input fields are wrapped in a single-form element🚀
- [ ] All input fields have the correct associated placeholder text.💯
- [ ] Message input has maxlength="500" attribute.🚀
- [ ] All input fields have the required attribute to activate the validation on submission.👌


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**